backend/src/interfaces/ICourses/IQueryCourses.ts

```diff
@@ -32,17 +32,16 @@ interface IUpdateCourseBody {
   title?: ICourse['title'];
   avatar?: ICourse['avatar'];
   description?: ICourse['description'];
-  skills?: {

```

Comment:
```Omit<ICourse, 'skills'>``` - is that what you need?

Reply:
Well, I could refactor this with `omit`, yeah. But we should specify here what actual fields we can update. So if we use `omit`, we will receive all the fields from course, even those we don't want to update